docs/source/api.md

```diff
@@ -16,14 +16,18 @@
 
 [[autodoc]] finish
 
-## save
-
-[[autodoc]] save
-
```

Comment:
It's not removed, just moved down to align with the order of `trackio.__init__.__all__`